rpcs3/CMakeLists.txt

```diff
@@ -183,18 +165,18 @@ elseif(WIN32)
     # share/cmake/Qt5/ for Qt5_Dir
 
     # If Qt5 is installed from official Qt installer
-    list(APPEND _QT5_TOOLS_PATHS "${Qt5_DIR}/../../../bin/")
+    # list(APPEND _QT5_TOOLS_PATHS "${Qt5_DIR}/../../../bin/")
 
     # If Qt5 is installed from vcpkg
-    list(APPEND _QT5_TOOLS_PATHS "${Qt5_DIR}/../../../tools/qt5$<$<CONFIG:Debug>:/debug>/bin/")
+    # list(APPEND _QT5_TOOLS_PATHS "${Qt5_DIR}/../../../tools/qt5$<$<CONFIG:Debug>:/debug>/bin/")
 
     add_custom_command(TARGET rpcs3 POST_BUILD
-        COMMAND set PATH=${_QT5_TOOLS_PATHS}$<SEMICOLON>%PATH%
+        # COMMAND set PATH=${_QT5_TOOLS_PATHS}$<SEMICOLON>%PATH%
         COMMAND "${CMAKE_COMMAND}" -E copy_directory "${CMAKE_SOURCE_DIR}/bin" "$<TARGET_FILE_DIR:rpcs3>"
         # If Qt5 is installed from vcpkg, add binary path to PATH
         # otherwise windeployqt tool won't be able to locate necessary dlls
-        COMMAND set PATH=${Qt5_DIR}/../../../$<$<CONFIG:Debug>:debug/>bin/$<SEMICOLON>%PATH%
-        COMMAND "${_WINDEPLOYQT}" --no-angle --no-compiler-runtime --no-opengl-sw --no-patchqt --no-translations --no-quick --plugindir "$<TARGET_FILE_DIR:rpcs3>/qt/plugins" $<$<CONFIG:Debug>:--debug> $<$<CONFIG:Release>:--release> "$<TARGET_FILE:rpcs3>")
```

Comment:
why is the plugindir removed

Reply:
Reverted. There was a problem with the switch, but I can't reproduce it anymore. I had to remove `$<$<CONFIG:Debug>:--debug> $<$<CONFIG:Release>:--release>` because `--release` leads to the error `Unable to find the platform plugin.` in MSYS2 and it works without those switches.